tests/cases/standalone/common/order/order_by.result

```diff
@@ -294,7 +294,7 @@ explain analyze select tag from t where num > 6 order by ts desc limit 2;
 +-+-+-+
 | 0_| 0_|_GlobalLimitExec: skip=0, fetch=2 REDACTED
 |_|_|_SortPreservingMergeExec: [ts@1 DESC] REDACTED
-|_|_|_SortExec: TopK(fetch=2), expr=[ts@1 DESC], preserve_partitioning=[true] REDACTED
```

Comment:
Is this a regression? We lost the topk hint.

Reply:
> Is this a regression? We lost the topk hint.

@waynexia says this hint is currently ok to remove